### PR TITLE
Refactoring for eliminate utils package #47

### PR DESCRIPTION
**Pull request by HemantJoshi11**
Refactoring  utils package to lib/file

### CODE REVIEW

File: analyzer/analyzer.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package analyzer
 
 import (
 	"fmt"
+	"github.com/rodrigo-brito/gocity/lib/file"
```
Comment (rodrigo-brito) [inline analyzer/analyzer.go:5]: please, reorder the import, keep it together with  `gocity/lib` import.

File: model/node.go [hunk 2]
```diff
@@ -2,11 +2,10 @@ package model
 
 import (
 	"fmt"
+	"github.com/rodrigo-brito/gocity/lib/file"
```
Comment (rodrigo-brito) [inline model/node.go:5]: Import order :+1:

**Review by rodrigo-brito — CHANGES_REQUESTED** [review]
Hi @vidu171, thank you for your contribution. Please, just fix the import order.

File: analyzer/analyzer.go (hunk 1 above)
Comment (vidu171) [reply analyzer/analyzer.go:5]: Hi, did you mean to tag @HemantJoshi11 , for these changes? because this pull request was made by them.

Comment (rodrigo-brito) [reply analyzer/analyzer.go:5]: Hi @vidu171, sorry for the mistake.

File: model/node.go (hunk 2 above)
Comment (HemantJoshi11) [reply model/node.go:5]: ok. I will fix

**Comment by rodrigo-brito** [comment]
Hi @HemantJoshi11, thank you

**Pull request merged**